opm/input/eclipse/Schedule/Group/GroupKeywordHandlers.cpp

```diff
@@ -179,46 +179,24 @@ void handleGCONPROD(HandlerContext& handlerContext)
         }
 
         const Group::ProductionCMode controlMode = Group::ProductionCModeFromString(record.getItem("CONTROL_MODE").getTrimmedString(0));
+
+        // Set the group limit actions. Item 7 (EXCEED_PROC) gives the general action, items 11-13 (WATER_EXCEED_PROCEDURE etc.)
+        // can override this for water, gas or liquid rate limits.
```

Comment:
just a comment, you can consider whether to adapt. 

in theory, it can also override the action for oil rate limit through FLD. It means the the action for oil limit might not be item 7, this was also one of the main reasons to have a dedicated variable for oil limit in the groupLimitAction.